project-addons/rma_scanner/models/crm_claim.py

```diff
@@ -1,30 +1,52 @@
 from odoo import api, models, fields, _
+from odoo.tools.safe_eval import safe_eval
+import json
 
 
 class CrmClaim(models.Model):
     _inherit = 'crm.claim'
 
+    location = fields.Char('Location')
+
     @api.multi
     def find_claim_by_number_using_barcode(self, barcode):
         barcode = barcode.replace("-", "/")
         barcode = barcode.replace("'", "-")
-        claim = self.search([('number', '=', barcode)], limit=1)
 
-        if claim:
-            if claim.stage_id.name != "Pendiente de recibir":
-                message = _("The {} is already received").format(claim.number)
-                self.env.user.notify_warning(message=message, sticky=True)
+        next_is_location = False
+
+        if barcode.startswith('RMA'):
+            claim = self.search([('number', '=', barcode)], limit=1)
+            if claim:
+                if claim.stage_id.name != "Pendiente de recibir":
```

Comment:
Si este móudlo lo usáis en Italia, esto puede ser un problema, ya que el camppo name es traducible y en cada idioma pondrá una cosa.

Reply:
Uy, pues en principio en Italia no lo usamos, pero no descarto que se pueda usar en un futuro, en la próxima modificación del módulo añadiré el arreglo.